src/index.js

```diff
@@ -9,24 +9,36 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-const { wrap } = require('@adobe/helix-shared');
+const wrap = require('@adobe/helix-shared-wrap');
+const { optionalConfig } = require('@adobe/helix-shared-config');
```

Comment:
@tripodsan I must say I was skeptical regarding the repo split, but it's nice to see it in action.